clang/test/CIR/CodeGen/builtins.cpp

```diff
@@ -56,3 +56,14 @@ int *test_std_addressof2() {
   // LLVM: [[RES:%.*]] = load ptr, ptr [[ADDR]], align 8
   // LLVM: ret ptr [[RES]]
 }
+
+extern "C" void *test_return_address(void) {
+  return __builtin_return_address(1);
+
+  // CIR-LABEL: test_return_address
+  // [[ARG:%.*]] = cir.const #cir.int<1> : !u32i
+  // {{%.*}} = cir.llvm.intrinsic "llvm.returnaddress" [[ARG]] : (!u32i) -> !cir.ptr<!void>
```

Comment:
Unrelated to this PR, but `cir.llvm.intrinsic "llvm.returnaddress"` sounds a bit redundant, `cir.llvm.intrinsic "returnaddress" would be more appropriate. Another example: `cir.llvm.intrinsic "llvm.aarch64.neon.sabd"` probably best expressed as just `cir.llvm.intrinsic "aarch64.neon.sabd"`. It'd be nice to clean this up at some point.

Reply:
Sure, make sense.
I might just introduce some code (temporarily before we fixed existing case) in LLVM lowering to prefix CIR intrinsics with "llvm" right before creation of LLVM::CallIntrinsicOp, and in CIR gen just use name without "llvm" prefix.